docs/open-api/openapi.yaml

```diff
@@ -477,7 +477,21 @@ components:
           - "TOPIC"
           - "ROLE"
           - "METALAKE"
-
+    metadataObjectTypeWithOwner:
```

Comment:
Since this parameter is only used in the owner, please move it to the `owner.yaml` file.

BTW, how about renaming to `metadataObjectTypeOfOwner` ?

Reply:
Ok

Reply:
> Since this parameter is only used in the owner, please move it to the `owner.yaml` file.

This comment has not been resolved

Reply:
Ok